### DISCUSSION TITLE
gRPC OTEL context problem

### DISCUSSION THREAD

**Discussion by kevinspacey** [roadrunner-server/roadrunner#1697] [Q&A] [answered]
Good day! I build application with roadrunner and use OTEL plugin. OTEL works nice for HTTP plugin, traceparent header helps build right context for application span. Problem starts with GRPS - there is no traceparent header, so i dont know how build right context - application and server span does not matches. How can i solve this problem?

**Comment by rustatian** [comment, marked as answer]
Hey @kevinspacey 👋🏻 
The `gRPC` plugin does not support OTLP headers at the moment 😭, but feel free to make a feature request, and since this is fairly easy to add, I'll include it in the upcoming `v2023.3`.
